.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -529,7 +559,7 @@ dynamodb_GetItem:
               snippet_tags:
                 - dynamodb.JavaScript.docClient.get
   services:
-    dynamodb: {GetItem}
+    dynamodb: {GetItem, ExecuteStatement, BatchExecuteStatement}

```

Comment:
This is possibly okay, although most languages are including PartiQL only in the scenario, so including these actions feels not quite right.

Reply:
Okay, done